mgit/README.md

```diff
@@ -0,0 +1,87 @@
+# MGit
+
+Tool for management the multi-repository architecture.
+
+## Instalation
+
+```bash
+npm install @cksource/mgit
+```
+
+## Usage
+
+Firstly, you need to create a configuration file `mgit.json`:
+
+```json
+{
+  "packages": "packages/",
+  "dependencies": {
+    "package/path": "github/url"
+  }
+}
+```
+
+* `packages` - the directory where dependencies will be cloned,
+* `dependencies` - list of dependencies.
+
+The following dependencies are proper and can be placed into configuration file:
+
+Package will be installed in `packages/@scope/package` from branch `develop`:
```

Comment:
Explain what `scope/package` means. And it should rather be `organization/repository`, because here it is GH URL.